tests/topotests/ospf6-topo1-vrf/r1/ospf6d.conf

```diff
@@ -0,0 +1,27 @@
+hostname r1
+log file ospf6d.log
+!
+debug ospf6 message all
+debug ospf6 lsa unknown
+debug ospf6 zebra
+debug ospf6 interface
+debug ospf6 neighbor
+debug ospf6 route table
+debug ospf6 flooding
+!
+interface r1-stubnet vrf r1-cust1
+ ipv6 ospf6 network broadcast
```

Comment:
Please add

```
 ipv6 ospf6 hello-interval 2
 ipv6 ospf6 dead-interval 10
```

To all interfaces with OSPFv3 neighbors to speed up the convergence.... The default takes 30 seconds while with this it takes ~10 seconds.